target_score.txt

```diff
@@ -1,8 +1,8 @@
 # Target score for scrapper.py:
-6
+8
 
 # Target score for pipeline.py:
-6
+8
```

Comment:
This one should be specified only after scrapper realisation. Just put 0 here for now.